packages/primitives/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@strapi/primitives",
```

Comment:
Should this be a private package for now?

Reply:
I don't think it needs to be, this is going into a feature branch so it will need to be public to release betas.